.github/workflows/build.yml

```diff
@@ -145,6 +149,14 @@ jobs:
         with:
           name: Bicep.LangServer
           path: ./src/vscode-bicep/bicepLanguageServer
+          
+      - name: Run unit tests
+        run: xvfb-run -a npm run test:unit
```

Comment:
Out of interest, why do you need `xvfb-run` for unit tests? Makes sense that you need it for e2e.

Reply:
That's because the unit tests invoked by `vscode-test`, but I you just reminded me that for unit tests that's not required because `vscode` is mocked...am going to change the `test:unit` npm script and remove `xvfb-run`.